src/main/java/io/anserini/collection/Iso19115Collection.java

```diff
@@ -113,6 +120,20 @@ public Document(JsonNode json) {
               .get("gco:CharacterString").asText();
       this.publish_time = json.get("gmd:MD_Metadata").get("gmd:dateStamp").get("gco:Date").asText();
       this.url = json.get("gmd:MD_Metadata").get("gmd:dataSetURI").get("gco:CharacterString").asText();
+      this.purpose = json.get("gmd:MD_Metadata").get("gmd:identificationInfo").get("gmd:MD_DataIdentification").get("gmd:purpose")
+                     .get("gco:CharacterString").asText();
+      this.supplInfo = json.get("gmd:MD_Metadata").get("gmd:identificationInfo").get("gmd:MD_DataIdentification").get("gmd:supplementalInformation")
```

Comment:
Would it make sense to change `gmd:MD_Metadata` and other keywords into `static final String` so we don't get hard-to-debug typos?

Reply:
At least for the "keys" that are used multiple times.

Reply:
Idts, because there are a lot of these possible keys and it can get really messy even if its just for keys that appear multiple times.

Reply:
In terms of hard-to-debug typos, java threw null pointer exceptions with the line number whenever a key is incorrect so debugging shouldnt be too difficult

Reply:
ok